config/symbols.us.stno4.txt

```diff
@@ -1,87 +1,12 @@
 NO4_pStObjLayoutHorizontal = 0x80180864;
 NO4_pStObjLayoutVertical = 0x80180938;
-NO4_EntityUpdates = 0x80180A0C;
-g_EInitObtainable = 0x80180B98;
-g_EInitParticle = 0x80180BA4;
-g_EInitSpawner = 0x80180BB0;
-g_EInitInteractable = 0x80180BBC;
-g_EInitCommon = 0x80180BE0;
-g_EInitDamageNum = 0x80180BEC;
-g_EInitSpearGuard = 0x80180C88;
-NO4_BackgroundBlockInit = 0x80180D04;
-NO4_RedDoorTiles = 0x80180D2C;
 g_WaterSounds = 0x80180EE4;
 g_WaterXTbl = 0x80180EE8;
 NO4_PrizeDrops = 0x80181928;
-UNK_Invincibility0 = 0x801819F4;
-g_testCollEnemyLookup = 0x80181A20;
-g_testCollLuckCutoff = 0x80181D40;
-g_testColluCoords = 0x80181D44;
-g_testCollvCoords = 0x80181D4C;
-g_testCollElementLookup = 0x80181D54;
-g_testColliFrames = 0x80181D68;
-g_testCollPrizeTable = 0x80181D74;
-g_jewelSwordDropTable = 0x80181DB4;
-g_eDamageDisplayClut = 0x80181DD0;
-g_SineTable = 0x80181DF8;
-g_bigRedFireballAnim = 0x80182250;
-explosionVariantSizes = 0x8018234C;
-greyPuff_rot = 0x8018235C;
-greyPuff_yVel = 0x8018236C;
-explode_yVel = 0x80182384;
-explode_startFrame = 0x8018239C;
-explode_lifetime = 0x801823A0;
-g_ESoulStealOrbAngles = 0x80182404;
-g_ESoulStealOrbSprt = 0x80182414;
-g_ESoulStealOrbAnim = 0x80182474;
-g_splashAspects = 0x80182510;
-g_SplashYMovement = 0x80182530;
-g_SurfacingYTbl = 0x80182550;
-g_SideWaterSpeeds = 0x8018255C;
-g_SideWaterAngles = 0x8018259C;
-g_SmallWaterDropVel = 0x801825AC;
-cutscene_data_ferryman_left = 0x80185354;
-cutscene_data_ferryman_right = 0x801853B7;
+cutscene_data_ferryman_right = 0x801853B7; // used:true
```

Comment:
This didn't get removed previously, but it does now.  Since it isn't actually defined in the code due to a weird alignment situation, but is still referenced as a symbol, it needs to be defined in this file (for now).